lib/Dialect/AIEVec/Transforms/ConvertVectorToAIEVec.cpp

```diff
@@ -174,16 +174,21 @@ static bool canFoldAIEShiftAndBroadcast(aievec::BroadcastOp op,
 }
 
 static aievec::MulElemOp
-createMulElemOpForI8(ConversionPatternRewriter &rewriter, Value lval,
-                     Value rval, VectorType srcType, unsigned bitWidth,
-                     Location loc) {
+createMulElemOpForI8OrBf16(ConversionPatternRewriter &rewriter, Value lval,
```

Comment:
I realize that Javier insisted on a specific name for this function, but it seems to me that down the road this function will handle more cases. Trying to encode the exact uses cases in the function name will become impractical. It seems to me the naming this function createMulElemAieML would then be more appropriate. I also would like to see a comment right before L176 that lists which cases this function tries to handle. Right now, I need to de-cypher the function body.